### PR TITLE
ENH: Added tutorial notebooks.

### PR DESCRIPTION
**Pull request by CaptainKanuk**
First in a batch of academic tutorial notebooks that I would like to start maintaining. This wiki will probably not be the final solution, but I would like to centralize the notebooks for now.

### CODE REVIEW

**Comment by twiecki** [comment]
This is great, will take a look soon.

**Comment by twiecki** [comment]
Do you want feedback on the content?

**Comment by twiecki** [comment]
`To actually compute β, we can use linear regression. We find the OLS best fit line for all points (ra,t,rb,t), where ra,t is the asset's returns for time t, and rb,t the same for the benchmark. The slope of this line is β, and the y-intercept is α.`
I'd add an image after this to clarify, something like this: http://people.duke.edu/~charvey/Classes/ba350_1997/capm/capm48.gif

**Comment by twiecki** [comment]
```X = sm.add_constant(X)
model = regression.linear_model.OLS(Y, X)
model = model.fit()```

There's a nicer interface to this which you might want to consider: http://statsmodels.sourceforge.net/devel/example_formulas.html

**Comment by twiecki** [comment]
```
X2 = np.linspace(X.min(), X.max(), 100)
Y_hat = X2 * beta + alpha

plt.scatter(X, Y, alpha=0.3) # Plot the raw data
plt.xlabel("r_b")
plt.ylabel("r_a")

plt.plot(X2, Y_hat, 'r', alpha=0.9)  # Add the regression line, colored in red
```

This could also be turned into a one-liner with `seaborn.jointplot(x, y, kind='reg')`

**Comment by twiecki** [comment]
```
# Draw many random points
X = np.random.rand(n)
Y = np.random.rand(n)

for i in range(n):
    
    if X[i]**2 + Y[i]**2 > 1:
        outside_circle += 1
    else:
        in_circle += 1
```

This could be vectorized: `X = np.random.rand((n, 2))` and then `np.mean(np.linalg.norm(X) > 1)`. The second line might be a bit too dense but you can also just iterate `for pt in X: np.linalg.norm(pt) > 1; ...`. I might have messed up the axis ordering somehow but you should be able to figure it out.

**Comment by CaptainKanuk** [comment]
This feedback is all helpful. With respect to the last two comments, I'm a little worried that the syntactic sugar might make the code less approachable to new people. This notebook should be understandable to a business school student who may not be super familiar with programming. As such I actually leaned towards the clunkier, but simpler approaches.

**Comment by twiecki** [comment]
Good points. I suppose there's a trade-off between understandability and starting them on the right path from the getgo but that's more philosophical and not worth elaborating on here :).
